src/protocols/rdp/guac_rdpdr/rdpdr_service.h

```diff
@@ -74,23 +74,43 @@ struct guac_rdpdr_device {
     int device_id;
 
     /**
-     * An arbitrary device name, used for logging purposes only.
+     * Device name, used for logging and for passthrough to the
+     * server.
      */
     const char* device_name;
 
     /**
-     * Handler which will be called when the RDPDR plugin is forming the client
-     * device announce list.
+     * The length of the device name, used for generating stream.
      */
-    guac_rdpdr_device_announce_handler* announce_handler;
+    int device_name_len;
```

Comment:
Is this useful within `guac_rdpdr_device`? Or is the length really only an internal consideration of the code generating `device_announce`?

Reply:
Not really - best to just make it local.

Reply:
Localized.